worker/worker.go

```diff
@@ -33,15 +33,18 @@ import (
 )
 
 const (
-	lockingPriorityRenew      = 100 // highest
-	lockingPriorityPriceTable = 95
-	lockingPriorityFunding    = 90
-	lockingPrioritySyncing    = 80
-
-	lockingDurationRenew      = time.Minute
-	lockingDurationPriceTable = 30 * time.Second
-	lockingDurationFunding    = 30 * time.Second
-	lockingDurationSyncing    = 30 * time.Second
+	lockingPriorityActiveContractRevision = 105 // highest
```

Comment:
`nit` we can just alter the other priorities right and keep 100 as the highest? this is kind of odd looking